anax-in-container/horizon-container

```diff
@@ -122,6 +122,9 @@ checkRequirements() {
 start() {
 	if [[ -n "$1" ]]; then
 		defaultFileMountArg="-v ${1}:/etc/default/horizon:ro"
+		source /etc/default/horizon
+	else
```

Comment:
this else is when `$1` is not defined, so why does it source `$1` ?

Reply:
also, you should add a comment to the `source` statements that says that it makes the variables in `/etc/default/horizon` available to *this* script, but none of its children (because the variables in the file aren't exported). That won't be obvious to future developers modifying this code.

Reply:
oh, also, in the case in which `$1` is set, it should source `$1` not `/etc/default/horizon`. (The latter is the location in the container, and we are still on the host.)

Reply:
so the net is that this section should be:```
	if [[ -n "$1" ]]; then
		defaultFileMountArg="-v ${1}:/etc/default/horizon:ro"
                 source "$1"
	fi
```